docs/v1/P4Runtime-Spec.mdk

```diff
@@ -1612,9 +1612,10 @@ The `ActionProfile` message includes the following fields:
 
 * `max_group_size`, an `int32` representing the maximum sum of all member
   weights within any given selector group, in the case of an Action Selector, or
-  0 for an Action Profile. PSA programs can use the `@max_group_size` annotation
-  to provide this value for Action Selectors. If the annotation is omitted, the
-  P4Info field will default to 0.
+  0 for an Action Profile. `max_group_size` is never larger than `size`.
```

Comment:
Can we phrase things a bit stronger: "is never larger" -> "must be no larger"?

Reply:
The main reason I went with this phrasing is that I mostly saw the imperative style of 'this is a fact' rather than 'make sure this is true' throughout the rest of the section. Though I now see several examples with "must", so sure!